.github/workflows/skip-validation.yml

```diff
@@ -0,0 +1,45 @@
+name: Validation
```

Comment:
Didn't know that we can set the same workflow name for several files. I thought it is going to be some `if-else` expressions withing the code 🤔

Reply:
Apparently it's a [documented workaround](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/troubleshooting-required-status-checks#handling-skipped-but-required-checks).